src/display/annotation_layer.js

```diff
@@ -1615,7 +1621,12 @@ class AnnotationLayer {
             parameters.div.appendChild(renderedElement);
           }
         } else {
-          parameters.div.appendChild(rendered);
+          if (element instanceof PopupAnnotationElement) {
+            // popup div mustn't be on top of an annotation
```

Comment:
Let's change this comment to the following since it's slightly more descriptive as to why this situation should not happen: `Popup annotation elements should not be on top of other annotation elements to prevent interfering with mouse events.`